packages/manager/modules/pci/src/projects/project/storages/databases/database/current-queries/current-queries.html

```diff
@@ -0,0 +1,120 @@
+                </oui-switch>
+                <label
+                    class="d-inline-block mx-2"
+                    data-ng-bind="($ctrl.showActiveConnections ? 'pci_databases_current_queries_switch_enable' : 'pci_databases_current_queries_switch_disable') | translate"
+                >
+                </label>
+            </oui-field>
+        </section>
+        <section class="col-sm-12 col-md-3">
+            <h6
+                class="my-0"
+                data-translate="pci_databases_current_queries_refresh_label"
+            ></h6>
+            <oui-field>
+                <oui-switch
+                    class="d-inline-block"
+                    name="idle-connections-switch"
+                    model="$ctrl.autorefreshPage"
+                    on-change="$ctrl.onPageAutoRefreshChange(modelValue)"
+                >
+                </oui-switch>
+                <label
+                    class="d-inline-block mx-2"
+                    data-ng-bind="($ctrl.autorefreshPage ? 'pci_databases_current_queries_switch_enable' : 'pci_databases_current_queries_switch_disable') | translate"
+                >
+                </label>
+            </oui-field>
+        </section>
+    </div>
+    <oui-datagrid
+        id="currentQueriesDataGrid"
+        class="mt-1"
+        page-size="25"
+        rows="$ctrl.displayQueries"
+        empty-placeholder="{{:: 'pci_databases_current_queries_datagrid_empty_placeholder' | translate }}"
+    >
+        <oui-datagrid-column
+            data-title="'pci_databases_current_queries_datagrid_column_query' | translate"
+            property="query"
+            type="string"
+            data-sortable
+        >
+            <text-accordion style="width:30vw" value="$value"></text-accordion>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title="'pci_databases_current_queries_datagrid_column_pid' | translate"
+            property="pid"
+            type="string"
+            data-sortable
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title="'pci_databases_current_queries_datagrid_column_duration' | translate"
+            property="queryDuration"
+            type="string"
+        >
+            <span data-ng-bind="$value  | number : 2"></span>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title="'pci_databases_current_queries_datagrid_column_database' | translate"
+            property="databaseName"
+            data-sortable
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title="'pci_databases_current_queries_datagrid_column_client_address' | translate"
+            property="clientIp"
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title="'pci_databases_current_queries_datagrid_column_application_name' | translate"
+            property="applicationName"
+            data-sortable
```

Comment:
We can add `data-filterable` and `data-searchable` as well for some of these columns